### PR TITLE
conditional tests tooltip added

### PR DESCRIPTION
**Pull request by ernestosiak**
Added new tooltip on left side panel.
## Description

Added new tooltip for conditionally passed tests on side panel - there was tooltip only for all passed tests.
## Motivation and Context
[Issue 375](https://github.com/Cognifide/aet/issues/375)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

**Comment by Jakub-Izbicki** [comment]
Tested, all is ok!

**Comment by ernestosiak** [comment]
changelog updated

File: report/src/main/webapp/app/layout/sidepanel/sidepanel.view.html [hunk 1]
```diff
@@ -116,7 +116,9 @@
             data-trigger="hover" data-container="body">{{sidepanel.testsStats.warning}}</span> /
       <span class="status passed" data-toggle="popover" data-content="Passed tests"
             data-trigger="hover" data-container="body">
-        {{sidepanel.testsStats.passed + sidepanel.testsStats.conditionallyPassed}}
+        {{sidepanel.testsStats.passed}} </span>
```
Comment (tkaik) [inline report/src/main/webapp/app/layout/sidepanel/sidepanel.view.html]: Why did you remove this addition here? I think that by design the number of displayed Passed tests should also include the number of conditional passed ("conditional pass" is a special case of "pass")

Comment (ernestosiak) [reply report/src/main/webapp/app/layout/sidepanel/sidepanel.view.html]: Sorry for that! Fixed!
